app/views/pages/_form_for_footer.html.haml

```diff
@@ -0,0 +1,35 @@
+- content_for :nav_secondary do
+  %li.nav-secondary__item.nav-secondary__item--right
+    = render 'comfy/admin/cms/sites/mirrors', object: @page
+
+- content_for :right_column do
+  = render 'form_save'
+  = render 'comfy/admin/cms/files/index'
+
+%div{data:{dough_component: 'StickyElements'}}
+  = form_for @page, as: :page,
+                    url: form_url,
+                    class: 'form',
+                    html: { multipart: true, autocomplete: 'off'},
+                    data: { dough_component: 'UnsavedChangesPrompt', dough_scheduler_form: true } do |form|
+
+    = render 'comfy/admin/cms/partials/page_form_before', object: form
+    .l-panel-content.l-panel-content--form
+      .l-constrained{ data: page_form_component(@page.new_record?, default: ['URLFormatter'], optional: ['MirrorInputValue']), 'data-dough-urlformatter-base-url' => '{pageType}/{slug}'}
+
+        = form.hidden_field :scheduled_on, data: {'dough-scheduler-on' => ''}
+
+        = render 'comfy/admin/cms/partials/page_form_inner', form: form
+
+    - if lookup_context.exists? "#{@page.layout.identifier}", ['pages/form_blocks'], true
+      = render partial: "pages/form_blocks/#{@page.layout.identifier}"
+    - else
+      = render partial: 'form_blocks'
+
+    = render 'scheduler', object: form
+    = render 'footer_bar', object: form
+    = render 'comfy/admin/cms/partials/page_form_after', object: form
+
+= render 'activity_log_box'
+= render 'insert_link_dialog'
+= render 'insert_image_dialog'
```

Comment:
This is equal to https://github.com/moneyadviceservice/cms/blob/master/app/views/pages/_form_for_home_page.html.haml